File: Volume 11 (1100 - 1199)/1113/1113.rb
#!/usr/bin/env ruby

a, b = Array.new(gets.split.map(&:to_i))
while a != b
    if b < a
        puts "Decrescente"
    else
        puts "Crescente"
    end
    a, b = Array.new(gets.split.map(&:to_i))
end

File: Volume 11 (1100 - 1199)/1113/1113.c
#include <stdio.h>
 
int main() {
    int x, y;
    while (scanf("%d %d", &x, &y) && x != y) {
        if (x < y) printf("Crescente\n");
        else if (x > y) printf("Decrescente\n");
    }
    return 0;
}